flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -537,6 +537,7 @@ public final void invoke() throws Exception {
 			afterInvoke();
 		}
 		catch (Exception invokeException) {
+			LOG.error("Invoke Error", invokeException);
```

Comment:
I think word "invoke" is specific to code and could confuse users.
It could be something like `"Error while running task {}", getTaskName()`.